### DISCUSSION TITLE
离线启动功能是否应该取消或仅仅作为调试功能？

### DISCUSSION THREAD

**Discussion by Creependerman** [Meloong-Git/PCL#1238] [提问] [answered]
离线启动功能明显违背了mojang的EULA，即使不移除这个功能也应该进行限制。
不过目前国内有些人尽皆知的事情，保留离线登录可能会更糟了（）

**Comment by WorldHim** [comment, marked as answer]
应该不会考虑。毕竟如果正版连不上服务器也只能使用离线启动。而且国内没有正版的玩家并不少，国内外使用离线登录的服务器也不少。
